index.html

```diff
@@ -281,7 +294,7 @@ <h3 class="item-title">Skills</h3>
                 </div>
             </div>
         </section>
```

Comment:
- [ ] Please adjust the positioning of the "Get in touch" button on the mobile view to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/166681760-f0bf38ee-ab7a-445c-b281-71f295baf8c1.png) | ![image](https://user-images.githubusercontent.com/62290545/166681595-1c440432-c2d4-4171-8a9b-de9e925d5794.png) |